src/main/scala/cromwell/engine/package.scala

```diff
@@ -112,6 +112,12 @@ package object engine {
   object ExecutionStatus extends Enumeration {
     type ExecutionStatus = Value
     val NotStarted, Starting, Running, Failed, Done, Aborted, Aborting = Value
+
+    implicit class EnhancedExecutionStatus(val status: ExecutionStatus) extends AnyVal {
+      def isTerminal: Boolean = {
+        Seq(Failed, Done, Aborted) contains status
+      }
```

Comment:
Cool, this is kind of a nice way to cope with the inherent lameness of Scala Enumerations.